app/src/main/java/at/aau/serg/activities/GameScreenActivity.kt

```diff
@@ -45,6 +53,20 @@ class GameScreenActivity : AppCompatActivity() {
             transaction.commit()
         }
 
+        val initialCards = intent.getSerializableExtra("cards", Array<CardItem>::class.java)
```

Comment:
This always throws me an error
java.lang.NoSuchMethodError: No virtual method getSerializableExtra

Reply:
This is probably because this version of  `getSerializableExtra` only works on the API version >= 33. I have added a check for this. Can you try it out on your device?

Reply:
now it works